ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1403,13 +1470,20 @@ def test_tomcat_filecheck_too_permissive(self, modify_permissions):
             assert check["kw"]["type"] == 'mode'
             assert check["kw"]["expected"] == '0660'
             assert check["kw"]["got"] == '0666'
-            assert (
-                check["kw"]["msg"]
-                == "Permissions of %s are too permissive: "
-                   "0666 and should be 0660"
-                % check["kw"]["path"]
-            )
-
+            if (parse_version(version) >= parse_version('0.6')):
```

Comment:
The message was changed in commit [aaf987be](https://github.com/freeipa/freeipa-healthcheck/commit/aaf987be) which is in freeipa-healthcheck 0.5 and 0.6. The check here should be
```
if (parse_version(version) >= parse_version('0.5'))
```

Reply:
Since there is a change in this method, can you also add TestIpaHealthCheckFileCheck to the temp run? Thanks.

Reply:
@flo-renaud , modified the version check here to 0.5.